acquia_lift.module

```diff
@@ -485,6 +504,16 @@ function acquia_lift_personalize_agent_type() {
 }
 
 /**
+ * Implements hook_personalize_wizard_steps_alter().
+ */
+function acquia_lift_personalize_wizard_steps_alter(&$steps) {
```

Comment:
Can we pass the current agent data in as context and then use it to limit this reports link to when the plugin is type acquia_lift_target?  I get a link on a personalize_target campaign right now but of course it doesn't show me any reports when I click on it.